src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,43 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int MIN_FIGURES = 5;
+    private static final double DEFAULT_VALUE = 10;
+    private static final String DEFAULT_COLOR = "white";
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+    private final Random random = new Random();
+
+    private double getRandomSide() {
```

Comment:
private methods should be after public ones

Reply:
ok